elated-egrets/bangazon-internal-site
# Rm delete training programs

## DESCRIPTION
added ability to delete a training program

## How to pull down
Commands

```
git fetch --all
git checkout rm-delete-training-programs
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. run tests to confirm all of them pass
1. navigate to the training programs and find one that has already happened
1. attempt to delete the training program, you should be brought back to the list view with the training program not deleted.
1. find a training program or create one that will happen in the future
1. try to delete it, you should again be taken back to the list view and see the training program is no longer there
1. try to poke around in other ways and break the code

## What you changed
added view and template for deleting a training program as well as tests to verify it works as intended

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Has a teammate approved the PR?

Then Ship It!
